### DISCUSSION TITLE
Pyinstaller & Pytorch

### DISCUSSION THREAD

**Discussion by zirid** [pyinstaller/pyinstaller#7831] [PyInstaller]
I was able to pack a small example app that includes pytorch and pyQt using Pyinstaller but the dist folder is 4GB .

All I am doing is printing pytorch version on QLabel !

` from PyQt5.QtWidgets import QMainWindow, QApplication, QLabel, QVBoxLayout,  QPushButton, QWidget
  from PyQt5.QtGui import QIcon
  import sys, os
  from torch import __version__`

Am I using Pyinstaller the wrong way ?

**Comment by rokm** [comment]
> All I am doing is printing pytorch version on QLabel !

And to do so you are importing two packages that contain large amount of binary extensions and binary dependencies that cannot be broken down.

**Reply by zirid** [reply]
I checked again and 3.99GB is the torch\lib folder that contains all libs and dlls .

**Comment by bmaxdk** [comment]
I agree that the libraries you're including can make the standalone executable quite large. Can you remove unnecessary libraries you are using? Analyze the `torch\lib` folder to see if there are any libraries that are not needed for your specific use case.
